content/django-vs-flask.md

```diff
@@ -0,0 +1,67 @@
+# Django or Flask which one is should you choose your project?
```

Comment:
I think there is an error in the title, the correct question would be:

which one should you choose for your project?